helios-system-tests/src/main/java/com/spotify/helios/system/SystemTestBase.java

```diff
@@ -1107,6 +1107,17 @@ protected static void removeContainer(final DockerClient dockerClient, final Str
       public Object call() throws Exception {
         try {
           dockerClient.killContainer(containerId);
+        } catch (DockerException e) {
+          //noinspection StatementWithEmptyBody
+          if ((e instanceof DockerRequestException) &&
```

Comment:
super minor: if you just `catch (DockerRequestException)` you can skip half this block